server/models/handlers/appointment.js

```diff
@@ -48,8 +48,35 @@ async function cancelWorkerAppointments(workerId, specificDate) {
     }
 }
 
+async function cancelSpecificAppointment(appointmentId) {
+    // If this method is somehow called without specifying values for the required parameters, false is returned.
+    if (_.isNil(appointmentId)) return false;
+    try {
+        const queryOutput = await appointmentModel.returnAppointment(appointmentId);
+        //Check if appt exists
```

Comment:
Hi Peter, 
I am a bit confused about what the following is doing. Why are we mapping it to studentId? I would assume if the appointment returned in line 55 is null, then the appointment does not exist. Thus, I would recommend checking to see if the appointment is `_.isNil()` or `_.isEmpty()`.

Reply:
Change has been made, and tested via postman successfully

Reply:
After npm testing issues, I am re-instating the mapping to student Id. The way I made the change to look at the whole appointment array instead of one element has broken the functionality and I will explain my process and goals through detailed comments.
![image](https://user-images.githubusercontent.com/45774697/100907969-f1b86900-3498-11eb-830e-ed0e13bbe3c8.png)